dev-lang/ispc/metadata.xml

```diff
@@ -15,6 +15,18 @@
 		<flag name="gpu">
 			Enable Level0 GPU support in ispcrt.
 		</flag>
+		<flag name="ispc_targets_ARM">
```

Comment:
Underscores are reserved for USE_EXPAND flags and are not valid local flags.

Reply:
That's why I originally used llvm_targets there. In the end this is just a compiler-frontend using the llvm-backend that adds one more target in XE.

Should I change it back to llvm_targets or try to get ispc_targets added to USE_EXPAND?

Reply:
Use `LLVM_TARGETS` when they match, and regular flags without grouping when they don't. `USE_EXPAND` for a single package is ugly, and should be avoided.